models/course_alert.py

```diff
@@ -170,3 +172,50 @@ def send_alert(self, sections):
 
         # TODO(david): Implement exponential backoff for retries
         return res.ok
+
+class EmailCourseAlert(BaseCourseAlert):
+    """Course alert using email notifications."""
+
+    user_id = me.ObjectIdField(
+        unique_with=BaseCourseAlert.BASE_UNIQUE_FIELDS)
+
+    TO_DICT_FIELDS = BaseCourseAlert.TO_DICT_FIELDS + ['user_id']
+
+    def __repr__(self):
+        return "<EmailCourseAlert: %s, %s, %s %s>" % (
+            self.course_id,
+            self.term_id,
+            self.section_type,
+            self.section_num,
+        )
+
+    def send_alert(self, sections):
+
+        _conn = boto.connect_ses(
+            aws_access_key_id=s.AWS_KEY_ID,
+            aws_secret_access_key=s.AWS_SECRET_KEY)
+
+        current_user = user.User.objects.get(id=self.user_id)
+
+        email_body = \
```

Comment:
Does it make sense to hard-code the email body? Why not read it from a static file and fill in the parameters? That way, we could use a templating engine to fill in the email body.